server/src/domain/smart-info/smart-info.service.spec.ts

```diff
@@ -69,6 +78,18 @@ describe(SmartInfoService.name, () => {
   });
 
   describe('handleTagImage', () => {
+    it('should do nothing if machine learning is disabled', async () => {
+      configMock.load.mockResolvedValue([{ key: SystemConfigKey.MACHINE_LEARNING_ENABLED, value: false }]);
+
+      await sut.handleClassifyImage({ id: '123' });
```

Comment:
It is, yes. I just tried to adapt to the existing tests. Since they had all been under `handleTagImage` I figured the intention was to just treat all methods similar. We can certainly update it, though!

Reply:
I'm not aware of if/why they would be grouped together before, but it's probably better to keep it consistent with one describe per method and one it per test imo.

Reply:
Agreed!